frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-treatment-status/radiogram-card-content-treatment-status.component.html

```diff
@@ -0,0 +1,24 @@
+<ng-container>
+    <h5 class="card-title">Behandlungsphase</h5>
+    <p class="card-text">
+        <ng-container [ngSwitch]="(radiogram$ | async)?.treatmentStatus">
```

Comment:
Instead of using the switch, you could also think about using the `treatmentProgressToGermanName` pipe (from `frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/utils/treatment-progress-to-german-name.pipe.ts`). But this would require a more generic text like "Eine neue Phase hat begonnen: {{ treatmentStatus | treatmentProgressToGermanName }}", as "Die Erstversorgung sichergestellt hat begonnen" does not make sense.

But I also like your more specific messages ("hat begonnen", "ist sichergestellt"), so its also fine for me if you decide to not use the pipe.

Reply:
I also thought about using a generic message, maybe reusing the display elements from the treatment behavior tab. However, to keep the mental load low for the signalers, I opted for prepared sentences.